recipe/meta.yaml

```diff
@@ -1,27 +1,25 @@
 {% set name = "PymageJ" %}
-{% set version = "0.1" %}
-{% set sha256 = "a18103bbb773b88542fd84f7dffd36d7310d9644ac44e9bc6f2d5b049aa52f33" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "e0b4e8266423e5d8648920219985bdd59d6f22a7e89bc34f0b40a473399aba3e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.zip
```

Comment:
Thanks for dropping this. Yeah we don't need this any more as Conda-Build got a bit smarter here a while back.